package.json

```diff
@@ -1,19 +1,22 @@
 {
   "name": "DiscordOverflow",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "src/index.js",

```

Comment:
I guess it should be src/main.js. Also, if possible you can add the ts pipeline to the start script as well